bin/tsc-watch-monorepo.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/authentication
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const build = require('../packages/build');
+const debug = require('debug')('build-watch');
+const chokidar = require('chokidar');
+const fs = require('fs');
```

Comment:
Any reason to not use `fs-extra`? We can use `ensureDir` (also aliased to `mkdirp`) so we don't need the below dependency. Plus async/await support.